binding.gyp

```diff
@@ -13,25 +13,21 @@
       "OTHER_CFLAGS": [
         "-std=c++14",
         "-stdlib=libc++",
-        "-Wall",
-        "-Werror"
+        "-Wall"
```

Comment:
Why remove `-Werror`?

Reply:
Deprecation warnings otherwise make it fail. It's either this or rewriting the addon entirely. This can wait next year.